corehq/toggles.py

```diff
@@ -1494,3 +1494,19 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_INTERNAL,
     namespaces=[NAMESPACE_OTHER],
 )
+
+
+MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER = StaticToggle(
+    'mobile_signup_redirect_ab_test_controller',
+    'Enable the ab test for telling mobile signups to use desktops. Set on the fly on registration if mobile',
+    TAG_PRODUCT
```

Comment:
Optional: what would you think of setting the namespace explicitly, like on the other toggle? I can never confidently remember what the default namespace (I think it's user, but that's counter-intuitive because the majority of new toggles are domain-specific).

Reply:
Seems reasonable